server/gql/models/aggregate/tests/purchasedProductsUtils.test.js

```diff
@@ -3,6 +3,7 @@ import moment from 'moment';
 import { redis } from '@services/redis';
 import { handleAggregateQueries, queryOptions, queryRedis } from '@gql/models/aggregate/purchasedProductsUtils';
 import { TIMESTAMP } from '@utils/constants';
+import * as utils from '@utils';
```

Comment:
we shouldn't need to * as utils. Just import utils from '@utils' should be good enough